app/LoginButton/index.js

```diff
@@ -19,6 +26,10 @@ const LoginButton = () => {
       console.log(`New Token: ${newToken}`);
       const parameters = getParameters();
       console.log(parameters);
+      const userInfo = await getUserInfo();
+      console.log('User Info', userInfo);
```

Comment:
```suggestion
      console.log(`User Info: ${userInfo}`);
```
Para que quede consistente con los logs anteriores, sino se nota que lo programaron dos personas distintas.